packages/builder/src/pages/builder/portal/apps/index.svelte

```diff
@@ -294,6 +314,10 @@
 </Modal>
 
 <AppLimitModal bind:this={appLimitModal} />
+<AccountLockedModal bind:this={accountLockedModal} onConfirm={() =>
+    isOwner
+      ? $licensing.goToUpgradePage()
+      : window.open("https://budibase.com/pricing/", "_blank")} />
```

Comment:
Could update this to be `$licensing.goToPricingPage()` so we can abstract this away, there are now a number of places we do this

Reply:
Done.